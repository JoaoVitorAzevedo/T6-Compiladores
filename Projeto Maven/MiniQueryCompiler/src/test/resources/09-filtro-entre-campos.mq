LOAD "data.json"
FILTER .idade < .valor