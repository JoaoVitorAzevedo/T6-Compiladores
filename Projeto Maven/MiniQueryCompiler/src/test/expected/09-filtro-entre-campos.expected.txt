✔ ANÁLISE SEMÂNTICA VÁLIDA

CÓDIGO JS GERADO:
// Código gerado automaticamente
const fs = require('fs');

const input = JSON.parse(fs.readFileSync('data.json', 'utf-8'));
const filteredData = input.filter(item => item.idade < item.valor);